packages/manager/apps/dedicated/client/app/dedicated/server/dedicated-server.routes.js

```diff
@@ -70,6 +71,11 @@ angular.module('App').config(($stateProvider) => {
       serverName: /* @ngInject */ $transition$ => $transition$.params().productId,
       specifications: /* @ngInject */ (serverName, Server) => Server.getBandwidth(serverName),
       user: /* @ngInject */ currentUser => currentUser,
+      vrack: /* @ngInject */ (
+        Server,
+        serverName,
+      ) => Server.getVrack(serverName)
+        .catch(() => null),
```

Comment:
No difference between each type of error ?

Reply:
We are stuck in an infinite loop 😅  (this question was already asked a lot this week, maybe if you do this intentionally, you could add a comment to say it's legit)